pkg/scheduler/framework/preemption/preemption.go

```diff
@@ -378,8 +378,8 @@ func (ev *Evaluator) prepareCandidate(ctx context.Context, c Candidate, pod *v1.
 				return
 			}
 		}
-		fh.EventRecorder().Eventf(victim, pod, v1.EventTypeNormal, "Preempted", "Preempting", "Preempted by %v/%v on node %v",
-			pod.Namespace, pod.Name, c.Name())
+		fh.EventRecorder().Eventf(victim, pod, v1.EventTypeNormal, "Preempted", "Preempting", "Preempted by a pod scheduled by %s on node %v",
```

Comment:
-1

The scheduler name is already part of the Event metadata:


```
  Type    Reason     Age   From                                   Message
  ----    ------     ----  ----                                   -------
  Normal  Scheduled  24s   gke.io/optimize-utilization-scheduler  Successfully assigned alpha/sample-job-9fq6c-khbf4 to gke-demo-default-pool-938ff563-473d
```

Reply:
Good point.

```
Events:
  Type    Reason     Age   From               Message
  ----    ------     ----  ----               -------
  Normal  Scheduled  19s   default-scheduler  Successfully assigned default/pause1 to node-1
  Normal  Preempted  4s    default-scheduler  Preempted by a pod scheduled by default-scheduler on node node-1
```

@mimowo it's good to remove the duplicated scheduler-name in event, and then update the cherry-pick PRs.

Reply:
Fixed, updated the cherry-pick PR by cherry-picking the fix, PTAL.